lib/legacy-logging.js

```diff
@@ -0,0 +1,764 @@
+/* Copyright (c) 2013-2015 Richard Rodger, MIT License */
+'use strict'
+
+var Util = require('util')
+var _ = require('lodash')
+var Patrun = require('patrun')
+var Jsonic = require('jsonic')
+var Eraro = require('eraro')
+var Common = require('./common')
+var Legacy = require('./legacy')
+
+// Shortcuts
+var arrayify = Function.prototype.apply.bind(Array.prototype.slice)
```

Comment:
In this case, that makes sense. Can we change the other cases, where files aren't moving?